client-libs/typescript/openai-legacy/index.ts

```diff
@@ -0,0 +1,81 @@
+import * as openPipeClient from "../codegen";
+import * as openai from "openai-legacy";
+import { version } from "../package.json";
+
+// Anything we don't override we want to pass through to openai directly
+export * as openAILegacy from "openai-legacy";
+
+type OPConfigurationParameters = {
+  apiKey?: string;
+  basePath?: string;
+};
+
+export class Configuration extends openai.Configuration {
+  public qkConfig?: openPipeClient.Configuration;
```

Comment:
lol probably shouldn't be `qkConfig` anymore. 😂

Reply:
Yeah, we should change that